app/components/member.tsx

```diff
@@ -19,40 +19,43 @@ const Member = ({
   github,
   site,
 }: MemberProps) => {
+  const splitRole = role?.split(/\s*\(\s*(.*)\s*\)\s*/);
```

Comment:
figure out how to style some formatting from prismic bc doing this for newlines is crazy (director of ops\n vice president is what this does)